en/08_Changelogs/6.0.0.md

```diff
@@ -215,6 +216,10 @@ We've upgraded from `intervention/image` v2 to v3. One of the main improvements
 
 If you are directly interacting with APIs from `intervention/image` in your project or module you should check out [their upgrade guide](https://image.intervention.io/v3/introduction/upgrade).
 
+### Symfony dependencies have been upgraded from v6 to v7 {#symfony}
+
+We've upgraded the various `symfony` dependencies from v6 to v7. If you are directly interacting with `symfony` components in your project or module you should check out the [symfony upgrade guide](https://symfony.com/doc/current/setup/upgrade_major.html).
```

Comment:
`symfony` is a bit weird... probably should either be "Symfony" or `symfony/*`

Reply:
We also shouldn't call out that upgrade guide, as it's geared towards people who are using symfony framework for everything in their project (the profiler for example probably won't work in a silverstripe app). Or if we do call it out, we should point out that some things in that guide won't work for a silverstripe app.

We didn't call out an upgrade guide for 5.0, probably don't need to here either.

Reply:
Updated